nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/GetMongo.java

```diff
@@ -60,23 +59,6 @@
 @InputRequirement(Requirement.INPUT_ALLOWED)
 @CapabilityDescription("Creates FlowFiles from documents in MongoDB")
 public class GetMongo extends AbstractMongoProcessor {
-    public static final Validator DOCUMENT_VALIDATOR = (subject, value, context) -> {
```

Comment:
@MikeThomsen I have replaced this one as well as `AGG_VALIDATOR` with the new `JSON_VALIDATOR`. I have ensured that the tests run fine. Also run a sample flow on a live MongoDB instance. Everything worked fine. Let me know, if you find anything odd.